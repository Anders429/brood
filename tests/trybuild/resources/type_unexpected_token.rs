use brood::Resources;

// Define resources.
struct A;
struct B;

type Resources = Resources!(A, + B);

fn main() {}
