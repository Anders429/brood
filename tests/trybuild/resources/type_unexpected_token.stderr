error: no rules expected the token `+`
 --> tests/trybuild/resources/type_unexpected_token.rs:7:32
  |
7 | type Resources = Resources!(A, + B);
  |                                ^ no rules expected this token in macro call
